### DISCUSSION TITLE
Batch img2img is using the same wildcard line of text. Possible to randomize it? (automatic1111)

### DISCUSSION THREAD

**Discussion by sevenstyles** [adieyal/sd-dynamic-prompts#104] [Q&A] [answered]
Hi, 

Thank you for such a useful script! I'm using batch img2img in automatic1111 with a wildcard but noticed it's using the same line of text from the wildcard for each image it creates.

Is it possible to have it select a random line of text from the wildcard file for each image processed through batch img2img?

**Comment by adieyal** [comment, marked as answer]
I'll take a look - i haven't used batch img2img before - fyi, I have created an issue for this here: https://github.com/adieyal/sd-dynamic-prompts/issues/107
